lib/examples/component-with-bindings.spec.ts

```diff
@@ -66,14 +66,11 @@ describe('component with bindings', () => {
   });
 
   it('displays the number of times the person was updated', async () => {
```

Comment:
I probably would have duplicated this test to have both scenarios covered. Either way, it does satisfy this scenario.